pages/ksphere/kommander/1.2/release-notes/index.md

```diff
@@ -10,75 +10,56 @@ enterprise: false
 
 <!-- markdownlint-disable MD034 -->
 
-# Release Notes for Kommander 1.2
+# Release notes for Kommander 1.2
+Kommander 1.2 was released on 11, November 2020 
```

Comment:
This still valid?

Reply:
I just cut GA tags so.. maybe `16, November 2020` instead?